app/classes/search_definition.rb

```diff
@@ -0,0 +1,40 @@
+class SearchDefinition
```

Comment:
Feels like what your `must`, `should`, etc. queries would be easier with https://github.com/rails/jbuilder?

Reply:
Or better yet, the ES DSL: https://github.com/elastic/elasticsearch-ruby/tree/master/elasticsearch-dsl